quantum-viz/pyproject.toml

```diff
@@ -24,6 +24,7 @@ python = "^3.6.1"
 click = "^8.0.1"
 ipython = "<8.0.0"
 varname = "^0.8.1"
```

Comment:
should we add jupyter notebook to the list, so you can see the example notebooks?

Reply:
I do not think so, this list should be minimal as possible